src/app/options.cpp

```diff
@@ -0,0 +1,394 @@
+/*
+ * Bittorrent Client using Qt and libtorrent.
+ * Copyright (C) 2014  Vladimir Golovnev <glassez@yandex.ru>
+ * Copyright (C) 2006  Christophe Dumez
+ *               2016  Eugene Shalygin
```

Comment:
Why do you love to violate the order? The copyright notice should be placed in reverse chronological order and be as follows:
```
Copyright (C) 2016  Eugene Shalygin <your_email>
```
You can see how it's done!